dev-support/Jenkinsfile

```diff
@@ -408,6 +410,9 @@ curl -L  -o personality.sh "${env.PROJECT_PERSONALITY}"
             // this needs to be set to the jdk that ought to be used to build releases on the branch the Jenkinsfile is stored in.
             jdk "JDK 1.8 (latest)"
           }
+          environment {
+            BASEDIR = "${env.WORKSPACE}/component"
```

Comment:
What's the difference between ${BASEDIR} and ${env.BASEDIR} [below](https://github.com/apache/hbase/pull/74/files#diff-070c3739c8fa8192e0741b4219d30de1R429)

Do they both read this value, or does only env.BASEDIR read this and the other one comes from ~magic

Reply:
env.BASEDIR gets replaced with the literal value by jenkins. BASEDIR is a bash variable that will be set to a value of the same thing that Jenkins would rewrite.

Reply:
this one doesn't have a node? so where does it exec?

Reply:
stages that don't have their own node label exec on the same node as the task that launches the entire pipeline (so the one that runs the "install yetus" and "init health results" steps)